src/index.js

```diff
@@ -166,6 +167,9 @@ function bind ({
   if (readyCallback) {
     source.once('value', readyCallback)
   }
+  if (errorCallback) {
```

Comment:
Yeah, that would work, give me some time to finish some work and I'll change this